docs/src/language_servers.md

```diff
@@ -512,11 +516,13 @@ There are multiple options:
     }
     ```
 
-### ruby-lsp
+### Ruby LSP
 
-1. Install the ruby-lsp gem (see [github:Shopify/ruby-lsp](https://github.com/Shopify/ruby-lsp)):
+1. Install the Ruby LSP gem (see [github:Shopify/ruby-lsp](https://github.com/Shopify/ruby-lsp)):
```

Comment:
Wouldn't be `ruby-lsp` better? It is the actual name of the gem. Also for the heading I'd personally at least leave the "-" in the name.

Reply:
Yeah, true, it should be `ruby-lsp` here.

But not in the heading, I would say. Look at the convention used on this page: https://lsp.sublimetext.io/language_servers/ . We tend to refer to servers using natural names and not their package/gem names (although there are exceptions of course).

Reply:
Alright, I changed it for sorbet & ruby-lsp.